src/empty-object.js

```diff
@@ -0,0 +1,5 @@
+export default function isEmptyObject(obj) {
+  for (let key in obj) return false
```

Comment:
I did, but then it also errored about the const being unused, maybe update the eslint rules to allow `_` as an unused? That way the line can be `for (const _ in obj) return false`